pluginsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -75,21 +89,29 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     interface IDeveloperIdProvider
     {
+        AuthenticationExperienceKind GetAuthenticationExperienceKind(); 
+
         String GetName();
 
         IIterable<IDeveloperId> GetLoggedInDeveloperIds();
 
+        // This API is deprecated and should not be used. The API will be removed once all callers have been moved over.
         Windows.Foundation.IAsyncOperation<IDeveloperId> LoginNewDeveloperIdAsync();
 
         void LogoutDeveloperId(IDeveloperId developerId);
 
         IPluginAdaptiveCardController GetAdaptiveCardController(String[] args);
 
+        // The below events (LoggedIn, LoggedOut and Updated) are legacy events.
         event Windows.Foundation.EventHandler<IDeveloperId> LoggedIn;
 
         event Windows.Foundation.EventHandler<IDeveloperId> LoggedOut;
 
         event Windows.Foundation.EventHandler<IDeveloperId> Updated;
+
+        event Windows.Foundation.TypedEventHandler<IDeveloperIdProvider, Object> Changed;
+
+        AuthenticationState developerIDState; 
```

Comment:
Capitalize?  And I assume this is intentionally a get/set property?

Reply:
Completing PR for now and resolving offline. I will be sure to take any changes in a future PR. Thanks!